ui/src/actions/user.js

```diff
@@ -106,7 +109,7 @@ export function userLocalLogin(credentials) {
       const response = await http.post('/accounts/login', credentials);
       dispatch(userLoginSuccess(response.data));
     } catch (error) {
-      const errorPayload = httpErrorToActionPayload(error)
+      const errorPayload = httpErrorToActionPayload(error);
```

Comment:
how this is not caught from the linter?

Reply:
it's prettier, apparently pre-commit doesn't always work properly :/